wgpu-core/src/command/ray_tracing.rs

```diff
@@ -1110,11 +1110,8 @@ fn iter_buffers<'a, 'b>(
                 wgt::IndexFormat::Uint16 => 2,
                 wgt::IndexFormat::Uint32 => 4,
             };
-            if mesh.index_buffer_offset.unwrap() % index_stride != 0 {
-                return Err(BuildAccelerationStructureError::UnalignedIndexBufferOffset(
-                    index_buffer.error_ident(),
-                ));
-            }
+            // since byte sizes of all the index formats are powers of two we could bitshift instead of multiplying
```

Comment:
```suggestion
```

This level of optimizing is basically never worth it, so I don't even want to entertain it, in this case too, the compiler just figures it out

https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=cecae298e18b6e59848035f6408e2259